ibm/resource_ibm_kms_key_policy.go

```diff
@@ -0,0 +1,476 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
```

Comment:
Change file name to  ibm/resource_ibm_kms_key_policy.go  ==>  ibm/resource_ibm_kms_key_policies.go
to be inline with resource name